src/Tile/Tile.test.js

```diff
@@ -89,6 +89,26 @@ describe('<Tile />', () => {
         </Tile>
     );
 
+    const productTile = (
+        <Tile className='pink' isButton
+            productTile>
+            <Tile.Media backgroundImage='https://techne.yaas.io/images/product-thumbnail-wide.png' />
```

Comment:
We should probably update these to an internal image since that one doesn't exist anymore or just a dummy url since nothing is getting rendered.